src/components/test/Calculator.test.js

```diff
@@ -0,0 +1,10 @@
+// import { Renderer } from 'react-test-renderer';
```

Comment:
@Alfred-KInyua and @ChrispaixK great work on your tests, kindly ensure you don't have commented coed in your codebase its not good practice.

Reply:
@danmainah Thanks for your review.

Reply:
Thank you @danmainah